pages/Category/index.jsx

```diff
@@ -76,9 +95,13 @@ class Category extends Component {
     return (
       <View title={this.title}>
         {this.props.isFilterBarShown && <FilterBar />}
-        {this.props.hasHeadline && this.createHeadline()}
+        {this.props.hasHeadline && this.renderHeadline()}
+        <Portal name={CATEGORY_LIST_BEFORE} id={this.id} />
```

Comment:
I think you can remove the `id` here. Or you'd need to refactor it to the new `props` attribute and add it to all portals of the page.